aspnetcore/grpc/loadbalancing.md

```diff
@@ -0,0 +1,334 @@
+---
+title: gRPC client-side load balancing
+author: jamesnk
+description: Learn how to make scalable, high-performance gRPC apps with client-side load balancing in .NET.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: jamesnk
+ms.date: 05/18/2021
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR], "service config"
```

Comment:
```suggestion
no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR, "service config"]
```

Reply:
The meta data build errors such as with author, etc, look fine to me.  I'm am guessing that what has tripped up the metadata section might be related to listing "service config" outside of the no-loc brackets.